aws-go uses dep

<!--
1. If you have a question and not a bug/feature request please ask it at http://forum.serverless.com
2. Please check if an issue already exists so there are no duplicates
3. Check out and follow our Guidelines: https://github.com/serverless/serverless/blob/master/CONTRIBUTING.md
4. Fill out the whole template so we have a good overview on the issue
5. Do not remove any section of the template. If something is not applicable leave it empty but leave it in the Issue
6. Please follow the template, otherwise we'll have to ask you to update it
-->

# This is a Bug Report

## Description
After creating an aws-go project, running `make build` fails due to missing Gopkg.toml

* What went wrong?
`make build` failed with error.

* What did you expect should have happened?
`make build` to succeed without dep dependency.

* What was the config you used?
The default project created using `serverless -t aws-go -p helloworld`

* What stacktrace or error message from your provider did you see?
No error from provider. Error came from `make build`. The error was

      dep ensure -v
      could not find project Gopkg.toml, use dep init to initiate a manifest
      make: *** [build] Error 1

Similar or dependent issues:
* #5144 
* #5145 

## Additional Data

* ***Serverless Framework Version you're using***: 1.30.0
* ***Operating System***: macOS High Sierra Version 10.13.5

--- a/lib/plugins/create/templates/aws-go/Makefile
+++ b/lib/plugins/create/templates/aws-go/Makefile
@@ -1,11 +1,10 @@
 build:
-	dep ensure -v
 	env GOOS=linux go build -ldflags="-s -w" -o bin/hello hello/main.go
 	env GOOS=linux go build -ldflags="-s -w" -o bin/world world/main.go
 
 .PHONY: clean
 clean:
-	rm -rf ./bin ./vendor Gopkg.lock
+	rm -rf ./bin
 
 .PHONY: deploy
 deploy: clean build

